docs/reference/action_list.rst

```diff
@@ -248,6 +248,17 @@ You can customize the list query thanks to the ``configureQuery`` method::
 Customizing the sort order
 --------------------------
 
+Default ordering in the list view
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default ordering may be defined in:
+
+1. ``$datagridValues`` property (of your Admin class)
+2. Query used to load data
+3. Defaults based on entity
+
+This order is important, because the first found will be used.
+
 Configure the default ordering in the list view
```

Comment:
Didn't know about this doc, this should be updated to recommend using getDefaultSortValues because datagridValues will be removed.